web/beacon-app/src/features/organizations/components/OrganizationTable/OrganizationTable.tsx

```diff
@@ -1,7 +1,7 @@
 import { Heading, Table, Toast } from '@rotational/beacon-core';
 import { useEffect, useState } from 'react';
 
-import { queryCache } from '@/config/react-query';
+import { queryCache } from '@/application/config/react-query';
```

Comment:
Note that I deleted the duplicate config module in `src` in favor of the config module in `src/application` as you requested.